deploy/examples/external/object-external.yaml

```diff
@@ -0,0 +1,19 @@
+#################################################################################################################
+# Create an object store with settings for replication in a production environment. A minimum of 3 hosts with
+# OSDs are required in this example.
+#  kubectl create -f object-external.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStore
+metadata:
+  name: external-store
+  namespace: rook-ceph # default set to rook-ceph
```

Comment:
Do you mean to use the `rook-ceph-external`?  didn't you recommended to use `rook-ceph` namespace instead of `rook-ceph-external`?  In cluster-external.yaml and common-external.yaml file, I have changed them to `rook-ceph `

Reply:
original object-external.yaml file has the same namespace as `rook-ceph`